Jenkinsfile-smoke.groovy

```
@@ -0,0 +1,27 @@
+/*
+* Requires: https://github.com/RedHatInsights/insights-pipeline-lib
+*/
+
+@Library("github.com/RedHatInsights/insights-pipeline-lib@v3") _
+
+if (env.CHANGE_ID) {
+    execSmokeTest(
+        // the service-set/component for this app in e2e-deploy "buildfactory"
+        ocDeployerBuilderPath: "xavier/xavier-analytics",
+    
+        // the service-set/component for this app in e2e-deploy "templates"
+        ocDeployerComponentPath: "xavier/xavier-analytics",
```

This must be changed to use the right one.
```suggestion
        ocDeployerComponentPath: "xavier/xavier-analytics-kieserver",
```

---

Agree! It comes from [here](https://github.com/RedHatInsights/e2e-deploy/tree/master/templates/xavier)

---

Thanks